Nightly - Update - Defaults - Show - Franchise Lists

Roving-Ronin opened the pull request:
<!--
    For Work In Progress Pull Requests, please use the Draft PR feature,
    see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

    For a timely review/response, please avoid force-pushing additional
    commits if your PR already received reviews or comments.

    Before submitting a Pull Request, please ensure you've done the following:
    - ✅ Test your changes locally to prove they work prior to submitting PRs.
    - 👷‍♀️ Create small PRs. In most cases this will be possible.
    - 📝 Use descriptive commit messages.
    - 📗 Update the CHANGELOG and Documentation where necessary.
-->

## What type of PR is this?

<!--
    Type X in the brackets of any relevant option, for example:
    - [X] Bug Fix (non-breaking change which fixes an issue)
-->
- [ ] Bug Fix (non-breaking change which fixes an issue)
- [X] Feature/Tweak (non-breaking change which adds new functionality or enhances existing functionality)
- [ ] Breaking Change (fix or feature that would break any existing functionality for users)
- [ ] Documentation Update
- [ ] Other 

## Description
Update the tv shows franchise yaml to:

Add:
- Spartacus franchise
- Death in Paradise franchise
- Reacher franchise

Update: 
- NCIS franchise (NCIS Origins & NCIS: Tony & Ziva)

## Related Issues [optional]

<!--
    For pull requests that relate or close an issue, please include them below. 
    We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

    For example having the text: "closes #1234" would connect the current pull request to issue 1234.
    And when the merged pull request reaches the master branch, Github will automatically close the issue.
-->
- Related Issue #
- Closes #

## Have you updated the Documentation to reflect changes (if necessary)?

<!--
    If your PR warrants Documentation changes, you are expected to make the relevant changes.
    Failure to do so will result in your PR not being approved.
    Type X in the brackets of any relevant option, for example:
    - [X] Yes
-->
- [ ] Yes
- [X] No

## Have you updated the CHANGELOG?

<!--
    Any PR that goes beyond a minor tweak (i.e. replacing a value, fixing a typo) should have a CHANGELOG entry.
    We may ask you to update the CHANGELOG if you haven't done so.
    Type X in the brackets of any relevant option, for example:
    - [X] Ye
-->
- [X] Yes
- [ ] No

Roving-Ronin committed:
Update franchise.yml - Add NCIS: Origins

Add NCIS: Origins to the addons: 4614 that defines all shows in the NCIS franchise.

Roving-Ronin committed:
Update franchise.yml

Add:
- Spartacus franchise
- Death in Paradise franchise
- Reacher franchise

Update: 
- NCIS franchise (NCIS Origins & NCIS: Tony & Zeva)

Roving-Ronin committed:
Update CHANGELOG - Update Defaults

Updates 'NCIS' list to include NCIS: Origins and NCIS: Tony & Zeva

Adds 'Spartacus', 'Death in Paradise' and 'Reacher' to the franchise lists

Roving-Ronin committed:
Update CHANGELOG

antwanchild commented:
Franchise is a dynamic collection pulled from tmdb

Roving-Ronin committed:
Update franchise.yml

Add:
      "60585": Bosch
      "3476": Inspector Morse
      "61511": Father Brown (2013)
      "41956": Death in Paradise (2011)

Roving-Ronin committed:
Update franchise.yml

Remove duplicate error

YozoraXCII commented:
> Franchise is a dynamic collection pulled from tmdb

This is true for Movies but no Shows, the Show franchises are manually maintained by us

YozoraXCII commented:
Could you update `.github/.wordlist.txt` with the following:

ncis
reacher
ziva

Roving-Ronin committed:
Update .wordlist.txt

As per: https://github.com/Kometa-Team/Kometa/pull/2594#issuecomment-2782688927

Roving-Ronin committed:
Update franchise.yml

YozoraXCII committed:
Merge branch 'nightly' into master
